cartography/data/jobs/analysis/azure_network_security_group_asset_exposure.json

```diff
@@ -6,43 +6,43 @@
 			"iterationsize": 100
 		},
 		{
-			"query": "MATCH (rule:AzureNetworkSecurityRule)<-[:CONTAIN]-(group:AzureNetworkSecurityGroup)<-[:RESOURCE]-(:AzureSubscription{id: $AZURE_SUBSCRIPTION_ID})<-[:RESOURCE]-(:AzureTenant{id: $AZURE_TENANT_ID})<-[:OWNER]-(:CloudanixWorkspace{id: $WORKSPACE_ID}) WHERE rule.access='Allow' AND rule.source_port_range='*' OR rule.protocol='*' OR rule.source_address_prefix='0.0.0.0/0' SET group.exposed_internet = true, (CASE WHEN NOT 'direct_ipv4' IN  coalesce(group.exposed_internet_type , []) THEN group END).exposed_internet_type = coalesce(group.exposed_internet_type , []) + 'direct_ipv4';",
+			"query": "MATCH (rule:AzureNetworkSecurityRule)<-[:CONTAIN]-(group:AzureNetworkSecurityGroup)<-[:RESOURCE]-(:AzureSubscription{id: $AZURE_SUBSCRIPTION_ID})<-[:RESOURCE]-(:AzureTenant{id: $AZURE_TENANT_ID})<-[:OWNER]-(:CloudanixWorkspace{id: $WORKSPACE_ID}) WHERE rule.access='Allow' AND rule.source_address_prefix IN ['0.0.0.0/0', '::/0', '*', 'Internet'] SET group.exposed_internet = true, (CASE WHEN NOT 'direct_ipv4' IN  coalesce(group.exposed_internet_type , []) THEN group END).exposed_internet_type = coalesce(group.exposed_internet_type , []) + 'direct_ipv4';",
```

Comment:
exposed_internet_type can't be ipv4. since we are checking for ipv6 as well :). we will have to make it dynamic based on source_address_prefix.